src/main/java/com/flawden/Calculator/controller/CalculatorController.java

```diff
@@ -0,0 +1,60 @@
+package com.flawden.Calculator.controller;
+
+import com.flawden.Calculator.service.CalculatorService;
+import org.apache.catalina.connector.Response;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+import java.util.UnknownFormatConversionException;
+
+@RestController
+@RequestMapping("/calculator")
+public class CalculatorController {
+
+    private CalculatorService calculatorService;
+
+    public CalculatorController(CalculatorService calculatorService) {
+        this.calculatorService = calculatorService;
+    }
+
+    @GetMapping()
```

Comment:
```suggestion
    @GetMapping
```
скобки пустые можно не писать